src/MediaPlayer/MPNowPlayingInfoCenter.cs

```diff
@@ -32,6 +32,16 @@ public MPNowPlayingInfo ()
 		public double? PlaybackDuration;
 		public double? DefaultPlaybackRate;
 
+		// iOS 10
```

Comment:
Yes for the availability attributes. I can still use properties on the new additions though.